float32 hunger_drive
float32 thirst_drive
float32 temp_drive
float32 adsign
float32 hsign
